### PR TITLE
update langchain agent with qwen2.5 for better accuracy

### PR DESCRIPTION
**Pull request by openvino-dev-samples**
image

### CODE REVIEW

**Comment by eaidova** [comment]
@openvino-dev-samples please update test patching:
```
  File "/tmp/scripts/patch_notebooks.py", line 204, in <module>
    patch_notebooks(args.notebooks_dir, args.test_device, args.skip_ov_install)
  File "/tmp/scripts/patch_notebooks.py", line 181, in patch_notebooks
    raise ValueError(f"Processing {notebookfile} failed: {source_value} does not exist in cell")
ValueError: Processing /tmp/notebooks/llm-agent-react/llm-agent-react-langchain.ipynb failed: mistralai/Mistral-7B-Instruct-v0.3 does not exist in cell
```

File: notebooks/llm-agent-react/llm-agent-react-langchain.ipynb [hunk 1]
```diff
@@ -279,22 +320,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 111,
    "id": "981df8fe-cfcf-455a-919e-dda36f3b5dfb",
    "metadata": {
     "test_replace": {
-     "mistralai/Mistral-7B-Instruct-v0.3": "TinyLlama/TinyLlama-1.1B-Chat-v1.0"
+     "Qwen/Qwen2.5-7B-Instruct": "TinyLlama/TinyLlama-1.1B-Chat-v1.0"
```
Comment (eaidova) [inline notebooks/llm-agent-react/llm-agent-react-langchain.ipynb]: Sometimes Qwen2.5-3B can not provide good results

Comment (eaidova) [reply notebooks/llm-agent-react/llm-agent-react-langchain.ipynb]:
```suggestion
     "Qwen/Qwen2.5-7B-Instruct": "Qwen/Qwen2.5-1.5B-Instruct"
```

**Comment by openvino-dev-samples** [comment]
I think a small model may lead parsing error on output of agent.

**Comment by eaidova** [comment]
>I think a small model may lead parsing error on output of agent.


@openvino-dev-samples ok, then I think it will be better to remove patching and disable notebook for precommit as we can not run 7B models due to github limitations (it is still will validated in internal infra with more powerful hw)

**Comment by openvino-dev-samples** [comment]
> > I think a small model may lead parsing error on output of agent.
> 
> @openvino-dev-samples ok, then I think it will be better to remove patching and disable notebook for precommit as we can not run 7B models due to github limitations (it is still will validated in internal infra with more powerful hw)

I'dont know if "TinyLlama/TinyLlama-1.1B-Chat-v1.0" can pass the CI, let me try

**Comment by eaidova** [comment]
> > > I think a small model may lead parsing error on output of agent.
> > 
> > 
> > @openvino-dev-samples ok, then I think it will be better to remove patching and disable notebook for precommit as we can not run 7B models due to github limitations (it is still will validated in internal infra with more powerful hw)
> 
> I'dont know if "TinyLlama/TinyLlama-1.1B-Chat-v1.0" can pass the CI, let me try

what about Qwen2.5-1.5B-Instruct? do you think it less accurate than tinyllama?

**Comment by openvino-dev-samples** [comment]
> > > > I think a small model may lead parsing error on output of agent.
> > > 
> > > 
> > > @openvino-dev-samples ok, then I think it will be better to remove patching and disable notebook for precommit as we can not run 7B models due to github limitations (it is still will validated in internal infra with more powerful hw)
> > 
> > 
> > I'dont know if "TinyLlama/TinyLlama-1.1B-Chat-v1.0" can pass the CI, let me try
> 
> what about Qwen2.5-1.5B-Instruct? do you think it less accurate than tinyllama?

It depends on how good LLM can follow prompt's instruction, Let me try ting-llama first

**Review by eaidova — APPROVED** [review]

**Pull request merged**